.github/CONTRIBUTING.md

```diff
@@ -66,6 +66,14 @@ This takes care of:
   - Release to PyPI via a [trusted publisher](https://docs.pypi.org/trusted-publishers/using-a-publisher/)
   - Automatically script updates in files where it's needed instead of hand-crafting it (i.e. in `pyproject.toml`)
 
+> [!CRITICAL]
```

Comment:
Should this one be a `[!WARNING]` or `[!CAUTION]`?

Reply:
Woops, onyl checked changes from a single commit. Disregard this comment.

Reply:
i am personally okay with this one